### PR TITLE
[Logging] More AI Logging Cleanup

### PR DESCRIPTION
**Pull request by Aeadoin**
Make logging similar between Bots, NPC, Mercs. less waypoint spam at General/Moderate logging levels.

### CODE REVIEW

File: zone/botspellsai.cpp [hunk 1]
```diff
@@ -1165,7 +1165,7 @@ bool Bot::AIDoSpellCast(uint8 i, Mob* tar, int32 mana_cost, uint32* oDontDoAgain
 				|| dist2 <= GetActSpellRange(AIBot_spells[i].spellid, spells[AIBot_spells[i].spellid].range)*GetActSpellRange(AIBot_spells[i].spellid, spells[AIBot_spells[i].spellid].range)) && (mana_cost <= GetMana() || GetMana() == GetMaxMana()))
 	{
 		casting_spell_AIindex = i;
-		LogAIModerate("Bot::AIDoSpellCast: spellid = [{}], tar = [{}], mana = [{}], Name: [{}]", AIBot_spells[i].spellid, tar->GetName(), mana_cost, spells[AIBot_spells[i].spellid].name);
+		LogAI("Bot::AIDoSpellCast: spellid = [{}], tar = [{}], mana = [{}], Name: [{}]", AIBot_spells[i].spellid, tar->GetName(), mana_cost, spells[AIBot_spells[i].spellid].name);
```
Comment (Akkadius) [inline zone/botspellsai.cpp:1168]: `[Bot::AIDoSpellCast]`

vs

`Bot::AIDoSpellCast:`

Comment (Akkadius) [reply zone/botspellsai.cpp:1168]: Remove commas and equals - the brackets show separation

File: zone/mob_ai.cpp [hunk 2]
```diff
@@ -364,21 +362,17 @@ bool NPC::AICastSpell(Mob* tar, uint8 iChance, uint32 iSpellTypes, bool bInnates
...
 			}
-#endif
 		}
 	}
 	return false;
 }
 
 bool NPC::AIDoSpellCast(uint8 i, Mob* tar, int32 mana_cost, uint32* oDontDoAgainBefore) {
-#if MobAI_DEBUG_Spells >= 1
 	LogAI("Mob::AIDoSpellCast: spellid = [{}], tar = [{}], mana = [{}], Name: [{}]", AIspells[i].spellid, tar->GetName(), mana_cost, spells[AIspells[i].spellid].name);
```
Comment (Akkadius) [inline zone/mob_ai.cpp:375]: All of these could be cleaned up if you're going through and actually cleaning up the logs

Remove equals, commas, prefix the function in brackets for consistency

File: zone/waypoints.cpp [hunk 3]
```diff
@@ -567,7 +567,7 @@ void NPC::SaveGuardSpot(bool ClearGuardSpot) {
 		if (m_GuardPoint.w == 0) {
 			m_GuardPoint.w = 0.0001; //hack to make IsGuarding simpler
 		}
-		LogAI("Setting guard position to {0}", to_string(static_cast<glm::vec3>(m_GuardPoint)));
+		LogAIDetail("Setting guard position to {0}", to_string(static_cast<glm::vec3>(m_GuardPoint)));
```
Comment (Akkadius) [inline zone/waypoints.cpp:570]: bracket

Comment (Akkadius) [reply zone/waypoints.cpp:570]: Remove slot `0`

File: zone/waypoints.cpp [hunk 4]
```diff
@@ -577,7 +577,7 @@ void NPC::SaveGuardSpot(const glm::vec4 &pos)
 
 	if (m_GuardPoint.w == 0)
 		m_GuardPoint.w = 0.0001;		//hack to make IsGuarding simpler
-	LogAI("Setting guard position to {0}", to_string(static_cast<glm::vec3>(m_GuardPoint)));
+	LogAIDetail("Setting guard position to {0}", to_string(static_cast<glm::vec3>(m_GuardPoint)));
```
Comment (Akkadius) [inline zone/waypoints.cpp:580]: bracket

File: zone/botspellsai.cpp (hunk 1 above)
Comment (Aeadoin) [reply zone/botspellsai.cpp]: Cleaned up.

File: zone/mob_ai.cpp (hunk 2 above)
Comment (Aeadoin) [reply zone/mob_ai.cpp]: Cleaned up.

File: zone/waypoints.cpp (hunk 3 above)
Comment (Aeadoin) [reply zone/waypoints.cpp]: Cleaned up.

File: zone/waypoints.cpp (hunk 4 above)
Comment (Aeadoin) [reply zone/waypoints.cpp]: Cleaned up.

**Review by Akkadius — APPROVED** [review]

**Pull request merged**